components/webui/imports/utils/misc.js

```diff
@@ -23,4 +23,67 @@ const computeHumanSize = (num) => {
     return `${Math.round(num)} B`;
 };
 
-export {computeHumanSize, sleep};
+/**
+ * Removes wrapping quotes from the given string, if it's quoted, and unescapes quotes from within
+ * the quoted string.
+ * NOTE: This method does *not* unescape non-quote characters, unlike most methods which handle
+ * unescaping quotes.
+ * @param str
+ * @param quoteChar
+ * @param escapeChar
+ * @return The processed string
+ * @throws Error if the quoted string has a quote within it (rather than at its ends) or it's
+ * missing one of it's begin/end quotes.
```

Comment:
```suggestion
 * @throws Error if any:
 *      - the quoted string has a quote within it (rather than at its ends)
 *      - it's missing one of it's begin/end quotes
 *      - the quoted / processed string is an empty one
```